hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java

```diff
@@ -945,15 +945,19 @@ public void usage(int exitCode) {
       o.printf("                       Here you can uniquely the relate-to descriptors by their\n");
       o.printf("                       owner ID / indicator-type / indicator-text, rather than\n");
       o.printf("                       by their IDs. See README.md for an example.\n");
+      o.printf("--reactions-to-add {...} Example for add/remove: INGESTED,IN_REVIEW\n");
+      o.printf("--reactions-to-remove {...}\n");
       o.printf("-c|--confidence {...}\n");
       o.printf("-s|--status {...}\n");
       o.printf("-r|--review-status {...}\n");
       o.printf("--first-active {...}\n");
       o.printf("--last-active {...}\n");
       o.printf("--expired-on {...}\n");
       o.printf("\n");
-      o.printf("Please see the following for allowed values in all enumerated types:\n");
+      o.printf("Please see the following for allowed values in all enumerated types except reactions:\n");
       o.printf("https://developers.facebook.com/docs/threat-exchange/reference/submitting\n");
+      o.printf("Please see the following for allowed values in all enumerated types in reactions:\n");
+      o.printf("https://developers.facebook.com/docs/threat-exchange/reference/reacting\n");
```

Comment:
Hmm, this definitely makes me miss python libraries for CLIs like argparse and click.

Reply:
There is definitely `getopt` ... there's a note elsewhere in the file about why i'm not using it here (fails to separate main-flags from verb-flags). For any CLI programs without verb-handlers, `getopt` is a great go-to, and (as a bonus) works largely the same across several languages.